compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -42,8 +42,8 @@ enum Frame<'a> {
 
 impl<'a> Frame<'a> {
     /// Construct a new frame around the delimited set of tokens.
-    fn new(tts: &'a [mbe::TokenTree]) -> Frame<'a> {
-        Frame::Delimited { tts, delim_token: token::NoDelim, idx: 0, span: DelimSpan::dummy() }
+    fn new(src: &'a mbe::Delimited, span: &DelimSpan) -> Frame<'a> {
```

Comment:
```suggestion
    fn new(src: &'a mbe::Delimited, span: DelimSpan) -> Frame<'a> {
```
`DelimSpan` is small and `Copy`, it can be passed by value here and in other places.